### DISCUSSION TITLE
要不要加入oas一起优化啊

### DISCUSSION THREAD

**Discussion by ohspecial** [lisai9093/YYS#24] [General]
大概看了下项目，挺好的，也推荐下oas
https://github.com/runhey/OnmyojiAutoScript
这个佬的项目挺好的，用模拟器比较安全，正常使用不会黑

**Comment by lisai9093** [comment]
等以后有空了了解下。目前没有新功能的计划。。On Nov 1, 2024, at 8:15 AM, ohspecial ***@***.***> wrote:﻿
大概看了下项目，挺好的，也推荐下oas
https://github.com/runhey/OnmyojiAutoScript
这个佬的项目挺好的，用模拟器比较安全，正常使用不会黑

**Reply by lisai9093** [reply]
今天抽空看了下，OAS貌似对跨平台支持不是很好。我目前主力机是macOS，所以跨平台最简单的方法还是放弃GUI，直接运行Python脚本。
